dialect/sql/sqlgraph/graph.go

```diff
@@ -84,6 +87,9 @@ type Step struct {
 	To struct {
 		// Table holds the table name of the neighbors (to).
 		Table string
+		// Schema is an optional name of the database
+		// where the table is defined.
+		Schema string
```

Comment:
You need to add the `Schema` field to the `Form` struct as well.

Reply:
@a8m from all the tests that I saw, the `Table` field in the `From` struct never actually ends up in the final SQL statement...This is mainly because the "From" comes from a `*sql.Selector` which already has a `sql.TableView` with the Table/Schema inside. 

I'm happy to ad the Schema field but I just wanted to note that in case I'm missing something :)

Reply:
You're right. Ignore this